src/core/actions.ts

```diff
@@ -59,36 +60,55 @@ export const setRequestStatusCode = assign<CoreContext, CoreEvents>({
     return (immutable.update(
       context.resquestsMetaDataById,
       requestId,
-      request =>
+      (request) =>
         immutable
           .wrap(request)
           .set('statusCode', statusCode)
-          .update('timeStamp', timeStamp => ({
+          .update('timeStamp', (timeStamp) => ({
             ...timeStamp,
-            end: new Date().getTime()
+            end: new Date().getTime(),
           }))
           .value()
     ) as unknown) as { [key: string]: CoreRequestMetaData };
-  }
+  },
 });
 
 export const setRequestHeaders = assign<CoreContext, CoreEvents>({
   resquestsMetaDataById: (context, event) => {
     const {
       requestId,
-      requestHeaders
+      requestHeaders,
     } = (event as ON_BEFORE_SEND_HEADERS).payload;
 
     if (!requestExist(context, requestId)) return context.resquestsMetaDataById;
 
     return (immutable.update(
       context.resquestsMetaDataById,
       requestId,
-      request => immutable.set(request, 'headers', requestHeaders)
+      (request) => immutable.set(request, 'headers', requestHeaders)
     ) as unknown) as { [key: string]: CoreRequestMetaData };
-  }
+  },
 });
 
 export const setSelectedRequest = assign<CoreContext, CoreEvents>({
-  selectedRequest: (_, event) => (event as OPEN_REQUEST_DETAILS).payload.request
+  selectedRequest: (_, event) =>
+    (event as OPEN_REQUEST_DETAILS).payload.request,
 });
+
+export const parseURLs = assign<CoreContext, CoreEvents>({
+  settings: (_, event) => {
+    if (!(event as SET_URLS).payload.urls.length) {
+      return { urls: [] };
+    }
+
+    return {
+      urls: (event as SET_URLS).payload.urls
+        .split(',')
+        .map((it) => it.replace(/\s/g, '').trim()),
```

Comment:
I like to add comments next to RegExps to make the reading process a bit easier. What do you think? A simple one, like `// Remove any whitespace`. It might help any future contributor. :rocket: 

Also, perhaps we can get rid of the `.trim()` call, as our RegExp is using `/g` to match all occurrences, including spaces surrounding the URL.